src/benchmarkRunner/BenchmarkRunner.ts

```diff
@@ -21,7 +21,7 @@ class BenchmarkRunner extends BenchmarkCollection {
 		this._comparisons.length = 0;
 	}
 
-	public async run(): Promise<void> {
+	public async run(isLogInCsvFormat: boolean): Promise<void> {
```

Comment:
NIT: we don't need to prefix the variable with "is". `logInCsvFormat` already indicates that it is a boolean value.